diff --git a/include/assimp/config.h.in b/include/assimp/config.h.in
index 3a6379b..380be7e 100644
--- a/include/assimp/config.h.in
+++ b/include/assimp/config.h.in
@@ -454,11 +454,10 @@ enum aiComponent
 };

 // Remove a specific color channel 'n'
-#define aiComponent_COLORSn(n) (1u << (n+20u))
+#define aiComponent_COLORSn(n) (1u << (n+16u))

 // Remove a specific UV channel 'n'
-#define aiComponent_TEXCOORDSn(n) (1u << (n+25u))
-
+#define aiComponent_TEXCOORDSn(n) (1u << (n+24u))
 // ---------------------------------------------------------------------------
 /** @brief Input parameter to the #aiProcess_RemoveComponent step:
  *  Specifies the parts of the data structure to be removed.

